CHANGELOG.md

```diff
@@ -17,10 +17,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Error responses of built-in endpoints are now JSON and follow the OData schema (#1919).
 - Code ids are now deleted rather than marked as `RETIRED`. `ACTIVE` is replaced with the more precise `ALLOWED_TO_JOIN`. (#1996)
 - CCF now depends on [Open Enclave 0.13](https://github.com/openenclave/openenclave/releases/tag/v0.13.0-rc2).
+- Snapshots are generated by default on the current primary node, every `10,000` committed transaction (#2029).
 
 ## Fixed
 
-- Scenario perf client `--msg-ser-fmt text` option works with C++ apps.
+- Scenario perf client `--msg-ser-fmt text` option works with C++ apps (#2026).
```

Comment:
Please remove this from the changelog. The perfclient is strictly an internal tool, it is not released, and there is no plan to change that or to support it.